packages/content/test/attributes/Website.test.ts

```diff
@@ -0,0 +1,93 @@
+import { ParsingError } from "@js-soft/ts-serval";
+import { Website } from "../../src";
+
+describe("creation of RepositoryAttributes of Attribute value type Website", function () {
+    test("can create a RepositoryAttribute of Attribute value type Website with the beginning www.", function () {
+        const validWebsite = Website.from({
+            value: "www.inwind.it"
+        });
+        expect(validWebsite.value.toString()).toBe("www.inwind.it");
+    });
+    test("can create a RepositoryAttribute of Attribute value type Website", function () {
+        const validWebsite = Website.from({
+            value: "https://inwind.it"
+        });
+        expect(validWebsite.value.toString()).toBe("https://inwind.it");
+    });
+    test("can create a RepositoryAttribute of Attribute value type Website with the German 'ä' in its domain", function () {
+        const validWebsite = Website.from({
+            value: "https://inwänd.it"
+        });
+        expect(validWebsite.value.toString()).toBe("https://inwänd.it");
+    });
+    test("can create a RepositoryAttribute of Attribute value type Website without www.", function () {
+        const validWebsite = Website.from({
+            value: "//google.it"
+        });
+        expect(validWebsite.value.toString()).toBe("//google.it");
+    });
+    test("can create a RepositoryAttribute of Attribute value type Website with an enhanced path", function () {
+        const validWebsite = Website.from({
+            value: "https://enmeshed.de/blog/meilenstein-enmeshed-als-komponente-ablage-in-mein-bildungsraum-geht-in-die-testphase-der-beta-version/"
+        });
+        expect(validWebsite.value.toString()).toBe(
+            "https://enmeshed.de/blog/meilenstein-enmeshed-als-komponente-ablage-in-mein-bildungsraum-geht-in-die-testphase-der-beta-version/"
+        );
+    });
+    test("returns an error when trying to create an Attribute value type Website with a blank in the value for Website", function () {
+        const invalidWebsiteCall = () => {
+            Website.from({
+                value: "https://inwind.test it"
```

Comment:
i think we are missing a test here with an encoded url like `https://inwind.test/%20it/`

Reply:
Good point, I changed this.